src/main/kotlin/no/nav/dagpenger/regel/api/streams/KafkaDagpengerBehovProducer.kt

```diff
@@ -3,103 +3,35 @@ package no.nav.dagpenger.regel.api.streams
 import mu.KotlinLogging
 import no.nav.dagpenger.events.Packet
 import no.nav.dagpenger.regel.api.APPLICATION_NAME
-import no.nav.dagpenger.regel.api.monitoring.HealthCheck
-import no.nav.dagpenger.regel.api.monitoring.HealthStatus
 import no.nav.dagpenger.regel.api.models.Behov
-import no.nav.dagpenger.streams.KafkaCredential
-import no.nav.dagpenger.streams.Topics
 import no.nav.dagpenger.streams.Topics.DAGPENGER_BEHOV_PACKET_EVENT
-import org.apache.kafka.clients.CommonClientConfigs
 import org.apache.kafka.clients.producer.KafkaProducer
-import org.apache.kafka.clients.producer.ProducerConfig
 import org.apache.kafka.clients.producer.ProducerRecord
 import org.apache.kafka.clients.producer.RecordMetadata
-import org.apache.kafka.common.KafkaException
-import org.apache.kafka.common.config.SaslConfigs
-import org.apache.kafka.common.config.SslConfigs
-import java.io.File
 import java.util.Properties
 import java.util.concurrent.Future
 
 private val LOGGER = KotlinLogging.logger {}
 
-internal fun producerConfig(
-    appId: String,
-    bootStapServerUrl: String,
-    credential: KafkaCredential? = null
-): Properties {
-    return Properties().apply {
-        putAll(
-            listOf(
-                ProducerConfig.BOOTSTRAP_SERVERS_CONFIG to bootStapServerUrl,
```

Comment:
Skulle vi ha utvidet dagpenger-streams til å kunne ta i mot dette i producerConfig?

Reply:
Dette er jo mange flere properties enn det vi har satt så langt i RiverConsumer

Reply:
Her settes jo acks til all, og compression til snappy, det har vi ikke gjort.

Reply:
Sant. Jeg endrer tilbake til de settingene som var.  Usikker på hvordan det kan/skal generaliseres. Ta in props som parameter som legges til?

Reply:
Det høres fornuftig ut :)